README.md

```diff
@@ -5,21 +5,57 @@ A website for making Animal Crossing tunes, written in OCaml with
 
 ## Development
 
+### Editor setup
```

Comment:
```suggestion
### Editor setup (Optional)
```

I'm a bit hesitant to start the readme with a large section of non-essential things to install (especially since I don't personally use these besides ocamlformat, and I imagine most people just want to know how to run the app). Maybe the editor setup section could be its own section at the bottom of the doc?

Reply:
Indeed.